src/pages/topology/ControlPlanes.js

```diff
@@ -53,19 +52,18 @@ class ControlPlanes extends Component {
     }
   }
 
-  componentWillMount() {
-    this.setState({loading: true});
+  componentDidMount() {
 
-    // Load overview for all templates
-    fetchJson('/api/v1/clm/model/cp_internal/CloudModel.yaml')
+    getInternalModel()
       .then((yml) => {
 
-        // Force a re-render
-        this.setState({
-          model: yml});
+        // Force a re-render if the page is still shown (user may navigate away while waiting)
+        if (this.refs.control_planes)
+          this.setState({
+            model: yml});
       })
       .catch((error) => {
-        // console.log(error);
+        console.log(error); // eslint-disable-line no-console
```

Comment:
It would be better to handle it instead of console log it. Because if there is an error here...it  stucks at '
Loading... Please wait' ...have to open console to see the error. Add an error banner or error message

Reply:
Good point. Will update with a fix